proposals/lip-0055.md

```diff
@@ -242,14 +242,16 @@ assetSchema = {
 
 #### Validation
 
-The general structure of the block assets is checked in the [static validation stage](#block-processing-stages) as follows:
+The general structure of the block assets is checked in the "static validation" and the "assets verification" [stage](#block-processing-stages) as follows:
 
 - **Static validation**:
-  - Check that each entry in the assets array has `module` corresponding to a module registered in the chain, while the `data` property has size at most equal to `MAX_ASSET_DATA_SIZE_BYTES`.
+  - Check that for each entry in the assets array the `data` property has size at most equal to `MAX_ASSET_DATA_SIZE_BYTES`.
   - Each module can insert at most one entry in the block assets. Hence, check that each entry has a distinct `module` property.
   - Check that the entries are sorted by lexicographical order of `module`.
+- **Assets verification**:
+  - Check that each entry in the assets array has `module` corresponding to a module registered in the chain.
 
-These validations are performed before the block is processed and without accessing the state. The individual entries are then checked in the assets verification verification stage.
+These validations are performed before the block is processed and without accessing the state. The individual entries are then also checked in the assets verification verification stage.
```

Comment:
```suggestion
These validations are performed before the block is processed and without accessing the state. The individual entries are then also checked in the assets verification stage.
```

Reply:
Above in the LIP we write

> Each module verifies the respective entry in the block assets. If any check fails, the block is discarded and has no further effect.

Would it make sense to just add this here as well as one bullet point and reformulate the sentence above?

Reply:
I changed this section. I'm not 100%ly if this is what you suggested.

Reply:
Looks all good.